JSTests/wasm/wasm.json

```diff
@@ -263,6 +267,8 @@
         "i31.new":             { "category": "gc",         "value": 251, "return": ["i31ref"],                       "parameter": ["i32"],                        "immediate": [], "extendedOp": 32 },
         "i31.get_s":           { "category": "gc",         "value": 251, "return": ["i32"],                          "parameter": ["i31ref"],                     "immediate": [], "extendedOp": 33 },
         "i31.get_u":           { "category": "gc",         "value": 251, "return": ["i32"],                          "parameter": ["i31ref"],                     "immediate": [], "extendedOp": 34 },
+        "array.get_s":             { "category": "gc",         "value": 251, "return": ["any"],                      "parameter": ["arrayref", "i32"],            "immediate": [{"name": "type_index", "type": "varuint32"}], "extendedOp": 20 },
```

Comment:
Minor nit: I think it'd be nice to have these just below `array.get` so it's ordered by opcode + have the columns aligned.

Reply:
Edit: actually, the other `wasm.json` does the former, the two json files need syncing I think.